dts/bindings/clock/st,stm32-hse-clock.yaml

```diff
@@ -13,3 +13,9 @@ properties:
     description: |
       HSE crystal oscillator bypass
       Set to the property to by-pass the oscillator with an external clock.
+
+  css-enabled:
+    type: boolean
+    description: |
+      HSE clock security system enabled
+      Set the property to enable the clock security system for the HSE clock.
```

Comment:
It could be nice adding some details on the effects.

Reply:
Thanks, I wasn't sure how verbose to be in the description. I've added more information, mostly taken from the reference manual's description.